clients/cobol-lsp-vscode-extension/src/services/CopybooksDownloader.ts

```diff
@@ -14,15 +14,21 @@
 
 import * as fs from "fs";
 import * as vscode from "vscode";
-import * as path from "path";
-import { loadDepFile, DependenciesDesc } from "./DependencyService";
-import { C4Z_FOLDER, DEPENDENCIES_FOLDER, PROCESS_DOWNLOAD_ERROR_MSG, DSN_NOT_FOUND_ERROR_MSG, INVALID_CREDENTIALS_ERROR_MSG, DSN_PLACEHOLDER, PROFILE_NAME_PLACEHOLDER, CONN_REFUSED_ERROR_MSG, NO_PASSWORD_ERROR_MSG } from "../constants";
-import { CopybookFix } from "./CopybookFix";
-import { CopybooksPathGenerator, createDatasetPath, createCopybookPath, checkWorkspace } from "./CopybooksPathGenerator";
-import { CopybookProfile, DownloadQueue } from "./DownloadQueue";
-import { ProfileService } from "./ProfileService";
-import { ZoweApi } from "./ZoweApi";
-import { Type, ZoweError } from "./ZoweError";
+import {
+    CONN_REFUSED_ERROR_MSG,
+    DSN_NOT_FOUND_ERROR_MSG,
+    DSN_PLACEHOLDER,
+    INVALID_CREDENTIALS_ERROR_MSG,
+    NO_PASSWORD_ERROR_MSG,
+    PROCESS_DOWNLOAD_ERROR_MSG,
+    PROFILE_NAME_PLACEHOLDER,
+} from "../constants";
+import {CopybookFix} from "./CopybookFix";
+import {checkWorkspace, CopybooksPathGenerator, createCopybookPath, createDatasetPath} from "./CopybooksPathGenerator";
+import {CopybookProfile, DownloadQueue} from "./DownloadQueue";
+import {ProfileService} from "./ProfileService";
+import {ZoweApi} from "./ZoweApi";
+import {Type, ZoweError} from "./ZoweError";
 
 export class CopybooksDownloader implements vscode.Disposable {
```

Comment:
I would prefer to avoid using -er endings in class names

Reply:
Main convention is class names should be noun. But any alternative name can be preferable, e.g.  CopybooksDownloadService or  CopybooksDownloadManager

Reply:
Yes, `CopybookDownloadService` would be better

Reply:
OK